apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/R4PatientResourceProvider.java

```diff
@@ -269,6 +269,13 @@ public Bundle searchByCoverageContract(
     List<Beneficiary> matchingBeneficiaries =
         fetchBeneficiaries(coverageId, includeIdentifiersValues, paging);
 
+    // Insures backwards compatability
+    boolean hasAnotherPage = matchingBeneficiaries.size() == paging.getPageSize();
```

Comment:
Definitely looks broken, and doesn't match the V1 logic.
```suggestion
    boolean hasAnotherPage = matchingBeneficiaries.size() > paging.getPageSize();
```

Reply:
fix that